groups-api/Groups API Sandbox (empty).postman_environment.json

```diff
@@ -0,0 +1,33 @@
+{
+	"id": "df1590e3-fcd6-4e88-8a3a-176e65aea4a0",
+	"name": "Groups API Sandbox (empty)",
+	"values": [
+		{
+			"key": "host",
+			"value": "https://api.sandbox.transferwise.tech",
+			"type": "default",
+			"enabled": true
+		},
+		{
```

Comment:
We can also add the refresh token so that we can add the `POST {{host}}/oauth/token` API endpoint.

Reply:
Added!